app/globals.css

```
@@ -77,3 +77,14 @@
     color: var(--color-foreground);
   }
 }
+
+@keyframes slideIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "slideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)